### ISSUE TITLE
Issue with Option 1 and TLS

### ISSUE THREAD

**Issue by JJGira** [MicrosoftDocs/OfficeDocs-Exchange#626]
I am trying to implement Option 1 in my work environment.  Everything is working fine except for one thing.  I only have TLS 1.2 enabled and this won't work.  When I re-enable TLS 1.0 it works.  What am I missing?  Thanks in advance...

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f1e4e3f0-bb9c-cca0-9343-49926262caba
* Version Independent ID: 75645bf5-f9a5-b5b9-d9e6-856e32ffc97f
* Content: [How to set up a multifunction device or application to send email using Office 365](https://docs.microsoft.com/en-us/exchange/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-3)
* Content Source: [Exchange/ExchangeOnline/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-3.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeOnline/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-3.md)
* Service: **exchange-online**
* Product: **exchange**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@JJGira do you feel the problem is on your end (your device) or on our end (Office 365)? Office 365 fully supports TLS 1.2, so could it be that your device only works with TLS 1.0?

**Comment by JJGira** [comment]
I am trying to configure this on a Windows 2012 R2 server.  Everything else is working fine with TLS 1.2 and the software we are using is supposed to work with TLS 1.2.  I will also verify that the software we are using supports it.

**Comment by scanum** [comment]
@JJGira Thank you very much for your feedback. Could you confirm that you have checked that the software you are using supports it?

**Comment by JJGira** [comment]
I'm sorry, when I posted this I stated I was having a problem with Option 1.  Guess I wasn't paying attention as I meant Option 2 (sending directly from the application).  Have been working with the vendor to find out where the problem is.  I can confirm that when TLS 1.0 is disabled, it doesn't work.  When TLS 1.0 is enabled, then it does work...  Trying to verify that the application does support TLS 1.2 only.  Thanks for following up...
